mcy/src/App.jsx

```diff
@@ -1,14 +1,25 @@
-import Stack from "@mui/material/stack";
-
 import Main from "./pages/Main";
+import BirthDay from "./pages/BirthDay";
+import Attendance from "./pages/Attendance";
+import News from "./pages/News";
+import ReaderAdmin from "./pages/ReaderAdmin.jsx";
+import MemberAdmin from "./pages/MemberAdmin.jsx";
 
 import "./App.css";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
 
 function App() {
   return (
-    <Stack>
-      <Main />
-    </Stack>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Main />} />
+        <Route path="/birthDay" element={<BirthDay />} />
+        <Route path="/attendance" element={<Attendance />} />
+        <Route path="/news" element={<News />} />
+        <Route path="/readerAdmin" element={<ReaderAdmin />} />
+        <Route path="/memberAdmin" element={<MemberAdmin />} />
```

Comment:
```suggestion
        <Route path="/reader-admin" element={<ReaderAdmin />} />
        <Route path="/member-admin" element={<MemberAdmin />} />
```
주소를 나타낼 때는 일반적으로 케밥 케이스로 나타냄. 
ReaderAdmin과 MemberAdmin이 뭘 나타내는지 알려줘
무슨 페이지를 말하는건지 스크린샷 첨부해주면 좋을듯

Reply:
@ykss ReaderAdmin은 셀 관리고 MemberAdmin 셀원 관리 페이지인데 주석 처리 같이 해서 다시 올릴게!